src/uwtools/config.py

```diff
@@ -70,6 +72,17 @@ def __init__(self, config_path=None):
 
         self.config_path = config_path
 
+        #initialize logger with parent inheritance
+        #pylint: disable=no-member
+        py_caller = getmodule(stack()[1][0])
+        if py_caller.__name__ in logging.root.manager.loggerDict:
```

Comment:
You have named this thing already in templater. To keep a strict contract, I think you need to just pass that name into this object and use that. Otherwise, we might start seeing some funky behavior if a caller decided to use a different naming convention.

Reply:
Thats fair, thanks! I was being overly committed to tier inheritance when I should just let the log entries track the modules.